Gems/WhiteBox/Code/Source/EditorWhiteBoxComponentMode.h

```diff
@@ -105,7 +99,7 @@ namespace WhiteBox
         AZ::Transform m_worldFromLocal;
         //! The function to use for querying modifier keys (while drawing).
         KeyboardModifierQueryFn
-            m_keyboardMofifierQueryFn;
+            m_keyboardModifierQueryFn;
```

Comment:
suggestion/nit: Can we please move this variable onto the same line as the type while we're here

```
KeyboardModifierQueryFn m_keyboardModifierQueryFn;
```